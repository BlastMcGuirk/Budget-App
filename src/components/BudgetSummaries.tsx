import React from 'react';
import { View } from 'react-native';
import BudgetSummary from '../components/BudgetSummary';
import { Layouts } from '../styles/global';

export default function BudgetSummaries() {
    return (
        <View style={Layouts.row}>
            <BudgetSummary
                budget='needs'
                remaining={1234.56}
                total={1500}/>
            <BudgetSummary 
                budget='wants'
                remaining={-123.45}
                total={500}/>
        </View>
    )
}
