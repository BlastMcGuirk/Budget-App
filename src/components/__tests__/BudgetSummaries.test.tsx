import renderer from 'react-test-renderer';
import BudgetSummaries from '../BudgetSummaries';

describe("Budget Summaries Tests", () => {

    it("Renders correctly", () => {
        const tree = renderer
            .create(<BudgetSummaries />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });

});