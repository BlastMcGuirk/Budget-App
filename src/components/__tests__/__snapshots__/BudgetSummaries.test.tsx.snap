// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Budget Summaries Tests Renders correctly 1`] = `
<View
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
      "justifyContent": "space-between",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "padding": 20,
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 24,
        }
      }
    >
      NEEDS
    </Text>
    <Text
      style={
        Array [
          Object {
            "fontSize": 36,
          },
          Object {
            "color": "green",
          },
        ]
      }
    >
      $
      1234.56
    </Text>
    <Text
      style={
        Object {
          "fontSize": 16,
        }
      }
    >
      / $
      1500
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "padding": 20,
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 24,
        }
      }
    >
      WANTS
    </Text>
    <Text
      style={
        Array [
          Object {
            "fontSize": 36,
          },
          Object {
            "color": "red",
          },
        ]
      }
    >
      $
      -123.45
    </Text>
    <Text
      style={
        Object {
          "fontSize": 16,
        }
      }
    >
      / $
      500
    </Text>
  </View>
</View>
`;
